enterprise/models.py

```diff
@@ -2,42 +2,114 @@
 """
 Database models for enterprise.
 """
-
 from __future__ import absolute_import, unicode_literals
 
-# from django.db import models
+from autoslug import AutoSlugField
+from simple_history.models import HistoricalRecords  # likely a bug in import order checker
+from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
-
+from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
 
 
+class EnterpriseCustomerManager(models.Manager):
+    """
+    Model manager for Enterprise Customer model.
+
+    Filters out inactive Enterprise Customers, otherwise works the same as default model manager.
+    """
+
+    # This manager filters out some records, hence according to the Django docs it must not be used
+    # for related field access. Although False is default value, it still makes sense to set it explicitly
+    # https://docs.djangoproject.com/en/1.10/topics/db/managers/#base-managers
+    use_for_related_fields = False
+
+    def get_queryset(self):
+        """
+        Return a new QuerySet object. Filters out inactive Enterprise Customers.
+        """
+        return super(EnterpriseCustomerManager, self).get_queryset().filter(active=True)
+
+
 @python_2_unicode_compatible
 class EnterpriseCustomer(TimeStampedModel):
     """
-    TODO: replace with a brief description of the model.
+    Enterprise Customer is an organization or a group of people that "consumes" courses.
+
+    Users associated with an Enterprise Customer take courses on the edX platform.
+
+    Enterprise Customer might be providing certain benefits to their members, like discounts to paid course
+    enrollments, and also might request (or require) sharing learner results with them.
+
+    Fields:
+        [implicit] id (IntegerField): EnterpriseCustomer identifier, PRIMARY KEY
+        name (CharField): Enterprise Customer name
+        short_code (AutoSlugField): Enterprise Customer code - used to reference this Enterprise Customer in other
+                                    parts of the system (SSO, ecommerce, analytics etc.)
+        active (BooleanField): used to mark inactive Enterprise Customers - implements "soft delete" pattern
     """
 
-    # TODO: add field definitions
+    class Meta(object):
+        verbose_name = _("Enterprise Customer")
+        verbose_name_plural = _("Enterprise Customers")
+
+    objects = models.Manager()
+    active_customers = EnterpriseCustomerManager()
+
+    name = models.CharField(max_length=500, blank=False, null=False, help_text=_("Enterprise Customer name."))
+    short_code = AutoSlugField(
```

Comment:
This is another comment that seems to have gone away. We should use UUID for values that need to be transmitted across systems. Short code is going to fail when we, inevitably, need to change it because a company changes its name. We can prevent these issues now by using UUID.

Again, 120 characters isn't exactly short.